framework/src/st_things/Kconfig

```diff
@@ -38,6 +38,21 @@ config ST_THINGS_ARTIK_HW_CERT_KEY
 	---help---
 		Build ST Things for STG Server
 
+config ST_THINGS_SECURESTORAGE
+	bool "Enable Secure Storage"
+	depends on ARCH_BOARD_ARTIK053 || ARCH_BOARD_ARTIK053S || ARCH_BOARD_ARTIK055 || ARCH_BOARD_ARTIK055S
```

Comment:
Please use ARCH_BOARD_ARTIK05X_FAMILY instead of ARCH_BOARD_ARTIK053 || ARCH_BOARD_ARTIK053S || ARCH_BOARD_ARTIK055 || ARCH_BOARD_ARTIK055S.

Reply:
Ok. Thanks alot.